src/patternfly/components/Alert/alert-icon.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="pf-c-alert__icon{{#if alert-icon--modifiers}} {{alert-icon--modifiers}}{{/if}}"
+  {{#if alert-icon--attributes}}
+    {{{alert-icon--attributes}}}
+  {{/if}}>
+  {{> @partial-block}}
+  {{#if pf-c-alert-icon--HasCheckCircleIcon}}
```

Comment:
change this to `alert-icon-success` same for the others. Just align with the state of the alert.

Reply:
done